website/react-magma-docs/src/pages/api/drawer.mdx

```diff
@@ -0,0 +1,66 @@
+---
+pageTitle: Drawer API
+title: Drawer
+props:
+  - DrawerProps
+---
+
+<DocsHeading type="api">Drawer</DocsHeading>
+
+A container with a transition
+
+## Basic Usage
+
+```typescript
+import React from 'react';
+import { Drawer } from 'react-magma-dom';
+
+export function Example() {
+  return <Drawer>Drawer content</Drawer>;
```

Comment:
These examples don't have a way to trigger the drawer.  Can you add a button like for the storybook stories?

Also, we should probably show an example where we set the drawer position.